src/components/FilterBadges/index.tsx

```diff
@@ -14,7 +15,9 @@ export const FilterBadges: React.FC = () => {
   const location = useLocation();
   const { search } = location;
   const queryParams = parseQueryString(search);
-  const queryParamsList = convertObjectToArray(queryParams);
+  const queryParamsList = convertObjectToArray(queryParams).filter(
+    ({ key }) => key !== ChartToggleQueryParams.chartType
```

Comment:
[EVG-14940]( https://jira.mongodb.org/browse/EVG-14940) will help improve this filter solution